plugins/coral-plugin-offtopic/client/components/OffTopicFilter.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styles from './styles.css';
+
+export default class OffTopicFilter extends React.Component {
+
+  tag = 'OFF_TOPIC';
+  className = 'offTopicComment';
```

Comment:
Same argument as above. If this is supposed to be a global css classname that can be overriden by users, consult david or riley for current naming convetions.